qualtran/surface_code/physical_parameters.py

```diff
@@ -0,0 +1,43 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import attr
+from attrs import frozen
+
+
+@frozen
+class PhysicalParameters:
+    t_gate = attr.ib(type=float, default=1e-6, repr=lambda x: f'{x:g}')  # 1us
+    t_meas = attr.ib(type=float, default=1e-6, repr=lambda x: f'{x:g}')  # 1us
```

Comment:
please use a variable name suffix to imply the unit. i.e. `t_gate_us`. Also consider using microseconds as the base unit